src/test/ui/in-band-lifetimes/ellided_lifetimes.rs

```diff
@@ -7,10 +7,13 @@
 // <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
+#![allow(warnings)]
+#![allow(unused_variables, dead_code, unused, bad_style)]
```

Comment:
also, you shouldn't need *both* allow warnings *and* the individual lints -- one ought to suffice, I would think

Reply:
True. I thought it might work with allow warnings but it doesn't for some reason 🤔